opengrok-web/src/test/java/org/opengrok/web/DirectoryListingTest.java

```diff
@@ -365,4 +373,14 @@ public void directoryListing() throws Exception {
             validateEntry((Element) nl.item(i));
         }
     }
+
+    @Test
+    public void directoryListingWithEftarException() throws IOException, HistoryException {
```

Comment:
The test looks weird without any assertion. It's not clear what is being tested. I'd definitely welcome a comment. I don't know of a nice way how to express in JUnit that the code should not throw an exception. I guess it's implied :D

Reply:
right. added verify()